### PR TITLE
update launch checklist to be clear on timing

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

File: pages/using-federalist/launch-checklist.md [hunk 1]
```diff
@@ -12,6 +12,6 @@ We've worked to make launching a live site on Federalist as simple as possible.
 1. Sign an IAA with 18F to use Federalist.
 2. Verify that your agency accepts the risk of launching a site on Federalist.
 3. Build your site and verify that the site is ready for launch using our preview URLs.
-4. Contact the Federalist team; we'll scan your site within a few days as part of our compliance process.
-5. Schedule launch of your custom URL. At your command, Federalist will initiate setup and provide you with a CloudFront URL such as https://d2xyasfn4889hb.cloudfront.net/. You will then configure DNS records with a CNAME to the CloudFront URL. Once this is complete, your site content will be available at your agency URL but with broken images. Technical background on this process is [here](/pages/how-federalist-works/custom-urls/).
+4. Contact the Federalist team; we'll scan your site within a few days as part of our compliance process and let you know of the results.
```
Comment (jseppi) [inline pages/using-federalist/launch-checklist.md]: Maybe add a "Any scan results of medium or higher severity must be cleared up before proceeding to launch." (or whatever the correct situation is).

Comment (wslack) [reply pages/using-federalist/launch-checklist.md]: Not a bad idea, but I think I'd rather tailor comms about these depending on who the partner is.

**Comment by jseppi** [comment]
merge?

**Comment by wslack** [comment]
Running behind.... and now merging

**Pull request merged**